### PR TITLE
Former

### PR DESCRIPTION
**Pull request by tharun143**

### CODE REVIEW

**Comment by tharun143** [comment]
@klumb This is PR #318 for issue #315 . I hope this is ok. If @germonprez suggests any names then I will them to the md file.

File: Community/gsoc-alumn.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# CHAOSS and Google Summer of Code Alumni
```
Comment (GeorgLink) [inline Community/gsoc-alumn.md:1]:
```suggestion
# CHAOSS Mentee alumni

We thank all CHAOSS mentees and mentors who participated in Google Summer of Code and Outreachy.
```

**Comment by GeorgLink** [comment]
> I would prefer to keep pull requests to one thing only: the board members and GSoC alumni pages should be different PRs.

I just saw that you did create two PRs. The issue may be that the origin of the`former` branch already had the `GSoC Alumni` commits.

File: About/former-board-members.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Former Board Members:
+
+* Jono Bacon
+* Jesús M. González-Barahona
+* Scott K. Peterson
```
[the remark this answers is not in the document]
Comment (klumb) [reply About/former-board-members.md:5]: Do we have that information? I know there was a board member change in August 2019. I believe there was another one prior but I don't remember when or who was involved.

**Comment by klumb** [comment]
@germonprez I have pictures for Jono, Jesus, and Josianne. I don't think we ever have one for Scott K. Peterson

Also, Josianne was missing from the list above. What is her last name?

**Comment by tharun143** [comment]
@GeorgLink I added the changes in the PR #321 
@klumb mention me in this issue when you found the pics of the board members. I will show an interest in helping.

**Comment by klumb** [comment]
@tharun143 please add Josianne Marsan to the list.

Also, I uploaded the images to the About folder. Please move them into an images subfolder and add the images to the markdown. We will need these image URL links to be full paths.

Thanks, we may need to experiment with image display keep an eye on it after it goes live to the website.

**Comment by tharun143** [comment]
@klumb but where is scott's photo?

**Comment by klumb** [comment]
He never provided us with one. Some people do not like their images posted on line. For him, we can say no image available.

**Comment by tharun143** [comment]
@klumb ok, fine

**Comment by tharun143** [comment]
@klumb I will send a PR from another branch as I recloned the repo I lost this branch.
Closing this PR.

**Pull request closed without merging**